src/api.js

```diff
@@ -17,6 +17,13 @@ export const fetchTopic = (topic) => {
         });
 };
 
+export const fetchSortOrder = (sort, order) => {
+    return api.get(`/articles?sort_by=${sort}&order_by=${order}`)
+        .then((res) => {
+            return res.data;
+        });
+};
+
 export const fetchSingleArticle = (article_id) => {
```

Comment:
you already have two api functions that are requesting a list of articles

There is a feature of axios that can help you out here 

The second argument to get (and other HTTP method methods) can be a config object like this:
`{ params: { <queryKey>: <queryValue> } }`
So our request might look like this:
```js
function getGuitars(guitarType) {
  return axios
    .get(`/api/guitars`, {
      params: {
        type: guitarType,
      },
    })
    .then((res) => res.data.guitars);
}
```
Now if guitarType is undefined it will be ignored and the request will just go to /api/guitars !

req_config docs: https://axios-http.com/docs/req_config
you can see it in an example here: https://axios-http.com/docs/example